presto-common/src/main/java/com/facebook/presto/common/QualifiedObjectName.java

```diff
@@ -61,10 +60,7 @@ public static QualifiedObjectName valueOf(String catalogName, String schemaName,
     @ThriftConstructor
     public QualifiedObjectName(String catalogName, String schemaName, String objectName)
     {
-        checkLowerCase(catalogName, "catalogName");
-        checkLowerCase(schemaName, "schemaName");
-        checkLowerCase(objectName, "objectName");
-        this.catalogName = catalogName;
+        this.catalogName = catalogName.toLowerCase(ENGLISH);
```

Comment:
I am struggling to understand, why only catalogName is lower cased and why did you remove the lower case check? Isn't it better to have a check than do always lower case.

Reply:
`checkLowerCase` is removed from `schemaNama` & `objectName` since that's not necessarily is lowercase now for all connector. At the places QualifiedObjectName is used, we are normalizing is based on connector.

Reply:
I am not an expert in this area, but what about `catalogName` ? can we retain the check for it?